ts/Series/GeoHeatmap/GeoHeatmapSeries.ts

```diff
@@ -0,0 +1,136 @@
+/* *
+ *
+ *  (c) 2010-2023 Highsoft AS
+ *  Authors:
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type GeoHeatmapSeriesOptions from './GeoHeatmapSeriesOptions.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+import GeoHeatmapPoint from './GeoHeatmapPoint.js';
+
+const {
+    seriesTypes: {
+        map: MapSeries
+    }
+} = SeriesRegistry;
+
+import U from '../../Core/Utilities.js';
+
+const {
+    extend,
+    merge
+} = U;
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * The Geo Heatmap series type.
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.geoheatmap
+ *
+ * @augments Highcharts.Series
+ */
+
+class GeoHeatmapSeries extends MapSeries {
+
+    /* *
+     *
+     *  Static Properties
+     *
+     * */
+
+    public static defaultOptions: GeoHeatmapSeriesOptions = merge(MapSeries.defaultOptions, {
+        keys: ['lon', 'lat', 'value']
```

Comment:
Use `pointArrayMap` instead of default `keys`. See other series type implementations for examples.